lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
@nmote I've add check for node because first line of flow is:
```
#!/usr/bin/env node
```
Some people use nvm for different versions of node without global installation(this installation is not in PATH of application, because app is not trigger .bashrc/.zshrc/etc). So this check is for this situations.